### PR TITLE
Added default polygon for virtual obstacle costmap layer.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by NadiaHG** [comment]
FR: Ok

**Comment by ipa-flg** [comment]
CR: Ok

**Comment by fmessmer** [comment]
where are these yaml's loaded? would it need to be added to a launch file?

(besides, is there a launch file from where the line path_segments and line_maps are loaded? we don't do this from anywhere within the bringup layer, right?)

**Comment by mgruhler** [comment]
Yep, this is not done from within bringup layer.

The maps are loaded in the respective navigation launch files.

**Comment by fmessmer** [comment]
Do we need empty `polygon_maps/restriction_polygon.yaml` files for all the other environments in order to not break launch file checks, i.e. automotive-assembly-line, empty, ipa-factory, ipa-kitchen?

If we receive one review (ok) from either @ipa-mig @ipa-flg @ipa-srd, I'll merge...

File: cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+restricted areas: 
+  - id: 31287680
```
Comment (floweisshardt) [inline cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml:2]: there should be no restricted area in the apartment. I assume this is a left over from testing. Please remove.

Comment (fmessmer) [inline cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml:2]: what does this parameter actually do?
does it 
1. restrict the robot to only move **within** the specified polygon or 
2. restrict the robot from moving **into** the specified polygon

Comment (stefandoerr) [inline cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml:2]: 2.

Comment (fmessmer) [inline cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml:2]: so where in the apartment is this?
do we have a visualization, e.g. screenshot from rviz?

Comment (stefandoerr) [inline cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml:2]: as @ipa-fmw wrote, this is a leftover from testing and shoud be removed

Comment (fmessmer) [inline cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml:2]: So we just close this PR without merging?

**Comment by floweisshardt** [comment]
+1 for closing without merging

**Comment by mgruhler** [comment]
@mig-em we need to discuss how to properly add an example of this once you are back...

**Pull request closed without merging**